vizro-core/changelog.d/20231004_131631_huong_li_nguyen_fix_child_forwardref.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+<!--
+### Changed
+
+- A bullet item for the Changed category.
+
+-->
+<!--
+### Deprecated
+
+- A bullet item for the Deprecated category.
+
+-->
+
+### Fixed
+
+- Fix bug on `ChildWithForwardRef.update_forward_refs()` having to be called explicitly ([#92](https://github.com/mckinsey/vizro/pull/92))
```

Comment:
```suggestion
- Fix bug so that `add_type` updates forward references in new type added ([#92](https://github.com/mckinsey/vizro/pull/92))
```

Reply:
I haven't read or understood our scheme for writing the changelog yet, so ignore me if this isn't in keeping with our usual tone... but usually a good rule of thumb is that the things we write should make some sense to the end user. Since `ChildWithForwardRef` doesn't exist outside our test suite, it won't mean anything to user.

Reply:
Yes, that makes a lot of sense! I like the phrasing as well 👍